libraries/lib-uuid/Uuid.h

```diff
@@ -0,0 +1,51 @@
+/*!********************************************************************
+
+ Audacity: A Digital Audio Editor
+
+ @file Uuid.h
+ @brief Declare a class to generate and parse UUIDs
+
+ Dmitry Vedenko
+ **********************************************************************/
+
+#pragma once
```

Comment:
#pragma once or follow the convention for #include guards?

Reply:
#pragma once is not standard

Reply:
I really think we should change our conventions. I've never seen a single compiler that had no support for pragma once